pom.xml

```diff
@@ -20,12 +20,12 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
     <!-- Web dependencies -->
-    <webjars-bootstrap.version>5.1.3</webjars-bootstrap.version>
+    <webjars-bootstrap.version>5.2.3</webjars-bootstrap.version>
```

Comment:
If you change this you should also change the templates where it is loaded in HTML (and probably regenerate the css as well?). Maybe leave this change for another PR?